.github/workflows/check-status.yml

```diff
@@ -47,7 +47,7 @@ jobs:
               const result = await github.rest.checks.listSuitesForRef({
                 owner: context.repo.owner,
                 repo: context.repo.repo,
-                ref: "${{ github.head_ref }}"
+                ref: "${{ github.event.pull_request.head.sha }}"
```

Comment:
this should be revert because multiple branch can have the same commit sha
that's why there was this fix : https://github.com/centreon/centreon/pull/5120